scripts/lib/CIME/XML/compilers.py

```diff
@@ -122,7 +122,7 @@ def get_value(self, name, attribute=None, resolved=True, subgroup=None):
         expect(subgroup is None, "This class does not support subgroups")
         value = None
 
-        node = self.get_optional_compiler_node(name)
+        node = self.get_optional_compiler_node(name, attributes=attribute)
```

Comment:
@jedwards4b would you take a close look at this? This seems like it could potentially have an impact on our system's behavior. I'm wondering if the ignoring of attributes was deliberate or just an oversight.